### ISSUE TITLE
NextJS yields - Error: The edge runtime does not support Node.js 'crypto' module.

### ISSUE THREAD

**Issue by jonahdc** [workos/workos-node#1085]
### Issue

While following the NextJS instructions at [User Management](https://workos.com/docs/user-management) docs, I encountered the  following error:

```
Error: The edge runtime does not support Node.js 'crypto' module.
Learn More: https://nextjs.org/docs/messages/node-module-in-edge-runtime
    at <unknown> (webpack-internal:///(middleware)/./node_modules/next/dist/esm/server/web/globals.js:34)
    at Object.get (webpack-internal:///(middleware)/./node_modules/next/dist/esm/server/web/globals.js:34:19)
    at new SubtleCrypto (webpack-internal:///(middleware)/./node_modules/@peculiar/webcrypto/build/webcrypto.es.js:2482:70)
    at new Crypto (webpack-internal:///(middleware)/./node_modules/@peculiar/webcrypto/build/webcrypto.es.js:2527:23)
    at eval (webpack-internal:///(middleware)/./node_modules/@workos-inc/node/node_modules/iron-session/dist/index.js:222:15)
    at (middleware)/./node_modules/@workos-inc/node/node_modules/iron-session/dist/index.js (file:///myproject/.next/server/middleware.js:2306:1)
    at __webpack_require__ (file:///myproject/.next/server/edge-runtime-webpack.js:37:33)
    at fn (file:///myproject/.next/server/edge-runtime-webpack.js:285:21)
    at eval (webpack-internal:///(middleware)/./node_modules/@workos-inc/node/lib/user-management/user-management.js:24:24)
    at (middleware)/./node_modules/@workos-inc/node/lib/user-management/user-management.js (file:///myproject/.next/server/middleware.js:2262:1)
    at __webpack_require__ (file:///myproject/.next/server/edge-runtime-webpack.js:37:33)
```

The `package.json` contains the following when running `npm install @workos-inc/authkit-nextjs`:

```
"@workos-inc/authkit-nextjs": "^0.8.1",
```
### Preliminary checks

I tried diagnosing the issue and it appears the introduction of `iron-session` version `~6.3.1` in version `7.15.0` is the main cause.

Here are some additional details:
1. `"iron-session": "~6.3.1"` [was introduced](https://github.com/workos/workos-node/pull/1078/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R40) in version `7.15.0` 
2. `"iron-session": "~6.3.1"` [has a dependency](https://github.com/vvo/iron-session/blob/v6.3.1/package.json#L143) on `"@peculiar/webcrypto": "^1.4.0"`
3. `"@peculiar/webcrypto": "^1.4.0"` [imports crypto](https://github.com/PeculiarVentures/webcrypto/blob/v1.4.3/src/crypto.ts#L1)

### Verification

Specifying a direct dependency on previous versions such as `"@workos-inc/node": "7.14.0"` resolves the issue.

----
Thanks!

**Comment by PaulAsjes** [comment]
Hi @jonahdc could you provide more details on how you're using the library? Are you using the `authkit-nextjs` library in your app or `workos-node`?

`authkit-nextjs` uses `iron-session` as a [dependency](https://github.com/workos/authkit-nextjs/blob/main/package.json/#L25) and works fine in Next edge runtimes, even when I revert it to v6.3.1.

**Comment by PaulAsjes** [comment]
Found the problem, we'll be putting up a fix for `authkit-nextjs` soon.

**Comment by PaulAsjes** [comment]
This is fixed in [v0.8.2](https://www.npmjs.com/package/@workos-inc/authkit-nextjs) of `authkit-nextjs`.

**Comment by jonahdc** [comment]
Thanks 💯
